articles/communication-services/quickstarts/email/includes/send-email-net.md

```diff
@@ -413,4 +413,36 @@ emailMessage.Attachments.Add(emailAttachment);
 
 You can download the sample app demonstrating this action from [GitHub](https://github.com/Azure-Samples/communication-services-dotnet-quickstarts/tree/main/SendEmailAdvanced/SendEmailWithAttachments)
 
+### Throw an exception when email sending tier limit is reached
```

Comment:
Should have a link to this section from the above section where the SendEmail samples are provided. And also from a troubleshooting page (Bala will know where this is).